include/cpu.h

```diff
@@ -103,8 +107,17 @@ typedef struct _cpu {
 
 cpu* mgec_new_cpu();
 
-cycles load8(cpu*, cpu_register_t, u8);
-cycles load16(cpu*, cpu_register_t, u16);
+cycles load8(cpu*, cpu_register8_t, u8);
+cycles load16(cpu*, cpu_register16_t, u16);
 #define load(C, R, V) _Generic((V), u8: load8, u16: load16)(C, R, V)
 
+u8 fetchReg8(cpu*, cpu_register8_t);
+u16 fetchReg16(cpu*, cpu_register16_t);
+
+void setReg8(cpu*, cpu_register8_t, u8);
+void setReg16(cpu*, cpu_register16_t, u16);
+
+cycles trans8(cpu*, cpu_register8_t, cpu_register8_t);
```

Comment:
Something I only just noticed when looking at the test again was: "Huh, why isn't Nono using trans directly?" And then I look here and there's no Generic mapping for trans at all. No intension of including a generic trans?

Reply:
No, because enum types aren't "unique" enough for `_Generic` to function (they're all just integers under the hood) - we need a unique discriminator to do with `_Generic`